src/user.js

```diff
@@ -32,6 +33,34 @@ class User {
       });
     });
   }
+
+  // Recipe to Cook Section //
+  addToRecipeToCook(recipe) {
+    if (!this.recipesToCook.includes(recipe)) {
+      this.recipesToCook.push(recipe)
+    }
+  }
+
+  removeFromRecipeToCook(recipe) {
+    const i = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(i, 1)
+  }
+
+  filterRecipeToCook(tag) {
```

Comment:
Should this be filterRecipesToCook? Because it's filtering through multiple recipes?